crates/rpc/rpc-types/Cargo.toml

```diff
@@ -25,10 +25,20 @@ serde_with = "3.3"
 serde_json.workspace = true
 jsonrpsee-types = { workspace = true, optional = true }
 alloy-primitives = { workspace = true, features = ["rand", "rlp"] }
+c-kzg = { workspace = true, features = ["serde"] }
+arbitrary = { workspace = true, features = ["derive"], optional = true }
```

Comment:
do we need arbitrary?
for what?

Reply:
removing—leftover from moving things around